readme.md

```diff
@@ -64,6 +64,12 @@ Type: `string`
 
 Capture a specific DOM element.
 
+##### css
+
+Type: `string`
+
+Apply CSS styles to the webpage.
+
```

Comment:
Added this block here, not sure if it should be added at the bottom of the options list? Not sure about the description either.